src/Core__Math.res

```diff
@@ -14,9 +14,19 @@ module Int = {
   @val external clz32: int => int = "Math.clz32"
   @val external imul: (int, int) => int = "Math.imul"
   @val external min: (int, int) => int = "Math.min"
-  @variadic @val external minMany: array<int> => int = "Math.min"
+  @variadic @val external _minMany: array<int> => int = "Math.min"
+  let minMany = arr =>
+    switch arr {
+    | [] => 0
+    | arr => _minMany(arr)
+    }
```

Comment:
There's no need to change the name of the external. I actually even prefer to use the same name for the override as the external, because the shadowing means the scope of the external is limited to only where it's needed. It's then immediately obvious that the function isn't used anywhere else, because it's not visible anywhere else.

Also, the use of underscore as prefix is especially problematic since that tells the compiler to not produce a warning if it's unused. That's a good warning to keep enabled.

Reply:
Done https://github.com/rescript-association/rescript-core/pull/82/commits/2219ebadb0a9ddba6e57f7065b9cab2622cefc55